dev-docs/modules/userid-submodules/pubprovided.md

```diff
@@ -23,18 +23,18 @@ The PubProvided Id module allows publishers to set and pass a first party user i
     });
     ```
 
-    Or, the eids values can be passed directly into the `setConfig` call:
+    Or, the eids values can be passed directly into the `setConfig` or `mergeConfig` call:
 
     ```javascript
-    pbjs.setConfig({
+    pbjs.mergeConfig({
```

Comment:
If there are multiple actors setting this config, they all should do the equivalent of:

```
            pbjs.mergeConfig({
                userSync: {
                    userIds: (() => {
                        const uidCfgs = pbjs.getConfig('userSync.userIds') || [];
                        let ppid = uidCfgs.find(cfg => cfg.name === 'pubProvidedId');
                        if (!ppid) {
                            ppid = {name: 'pubProvidedId', params: {eids: []}};
                            uidCfgs.push(ppid);
                        }
                        ppid.params.eids.push({
                            source: "example.com",
                            uids: [{
                                id: "example",
                                atype: 1,
                                ext: {
                                    stype: "ppuid"
                                }

                            }]
                        })
                        return uidCfgs;
                    })()
                }
            })
            pbjs.refreshUserIds({submoduleNames: ['pubProvidedId']})

```

Now that I wrote it down I am doubling down on the opinion that it's clearly a hack and not desirable as the default way to set up pubProvidedId - but I'll let you figure out the wording for it.

Reply:
we could also update the userId module to do this merging (so that a simple `mergeConfig` would work); it's not well defined in general, but it could be done as a special case for pubProvidedId, since it consists only of a list of EIDS. currently the first entry in `userIds` that has `name === 'pubProvidedId'` is the one that applies, so it shouldn't be breaking to start picking up subsequent ones.

Reply:
This PR as it is does not address the problem that prompted it (https://github.com/prebid/Prebid.js/issues/10548). On the other hand my example is quite convoluted and clearly a hack - this module is not meant to do this, so I'm not sure on how to proceed. Maybe we should put it in a separate section at the end? I think it's of interest to a very small number of people.

Reply:
perhaps the changes look better now?

Reply:
I'm fine with this 👍

I will merge and we'll wait for feedback